docker-compose.yml.example

```diff
@@ -9,6 +9,8 @@ services:
       dockerfile: saga.Dockerfile
     environment:
       - KEYNAME=relayer
+      - PROVIDER_CHAIN=
```

Comment:
We shouldn't use provider/consumer terminology here as that is only associated with CCV, which we don't currently have